snap/snapcraft.yaml

```diff
@@ -12,22 +12,13 @@ grade: devel
 confinement: strict
 
 apps:
-  run: 
-    command: server
+  cli: 
```

Comment:
the app having the name of the snap is the default one, so commands would be reduced to "wifi-connect <params>"